amundsen_application/static/js/ducks/search/reducer.ts

```diff
@@ -30,17 +34,29 @@ export function executeSearch(term: string, pageIndex: number): ExecuteSearchReq
 export type SearchReducerAction = ExecuteSearchRequest | ExecuteSearchResponse;
 
 export interface SearchReducerState {
-  pageIndex: number;
-  searchResults: SearchListResult[];
   searchTerm: string;
-  totalResults: number;
+  dashboards?: DashboardSearchResults;
```

Comment:
Is the intent to make `dashboards` and `users` required once we have them?

Reply:
That was the intent, but technically we can make these required already since we have the initial state that doesn't get changed.